### PR TITLE
Adding wait functionality

### PR DESCRIPTION
**Pull request by NimaSoroush**
- Wait step been added

### CODE REVIEW

File: src/chromyRunner.js [hunk 1]
```diff
@@ -68,6 +69,24 @@ const run = async (chromy, options, test) => {
           return false;
         }
         break;
+      case actions.wait:
+        try {
+          if (check.number(action.value)) {
+            logger.log(`Waiting for ${action.value} ms`);
```
Comment (danrr) [inline src/chromyRunner.js]: use `prefixedLogger` here to get the test name, it's defined above

Comment (NimaSoroush) [inline src/chromyRunner.js]: true

Comment (danrr) [inline src/chromyRunner.js]: worth making the messages lower-case as I've changed the others to that

Comment (NimaSoroush) [inline src/chromyRunner.js]: true again. doing refactoring here

**Pull request merged**